From 150d752b8e76c25b4e308b5fd402ec4738529848 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Mon, 25 Mar 2019 16:42:06 +0800
Subject: [PATCH 236/245] arm64: dts: define big endian of IFC NOR for LS1043a

Integrated flash controller present of NOR in LS1043A is big endian.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb.dts | 1 +
 1 file changed, 1 insertion(+)
 mode change 100644 => 100755 arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb.dts

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb.dts
old mode 100644
new mode 100755
index 09765dc4f2ff..99e73a803445
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb.dts
@@ -70,6 +70,7 @@
 			reg = <0x0 0x0 0x8000000>;
 			bank-width = <2>;
 			device-width = <1>;
+			big-endian;
 		};
 
 		nand@1,0 {
-- 
2.17.1

